Makefile

```diff
@@ -41,8 +44,14 @@ binary-local:
 build-container:
 	docker build ${DOCKER_BUILD_ARGS} -t "$(DOCKER_IMAGE)" .
 
+docs/%.1: docs/%.1.md
+	$(GO_MD2MAN) -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
+
+.PHONY: docs
+docs: $(MANPAGES_MD:%.md=%)
+
 clean:
-	rm -f skopeo
+	rm -f skopeo docs/*.1
 
 install: install-binary
 	install -m 644 man1/skopeo.1 ${MANINSTALLDIR}/man1/
```

Comment:
@lsm5 I'm converting from a man formatted file to a markdown formatted file.  Therefore, I don't think these changes are desired.